ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -1023,3 +1023,20 @@ ethernet_interfaces:
       frequency_unit: ghz
       media:
         override: 100gbase-ar4
+  - name: Ethernet70
+    description: dot1x_aaa_unresponsive
+    shutdown: false
+    dot1x:
+      aaa:
+        unresponsive:
+          eap_response: success
```

Comment:
eap response config is not generated in cfg file, please take look

Reply:
Fixed.